src/main/java/stream/flarebot/flarebot/FlareBot.java

```diff
@@ -145,6 +151,8 @@
     private static String statusHook;
     private static String token;
 
+    private static OkHttpClient client = new OkHttpClient.Builder().connectionPool(new ConnectionPool(4, 10, TimeUnit.SECONDS)).build();
+
```

Comment:
Put a user agent, specifically `Mozilla/5.0 FlareBot`
First part is used to make CloudFlare leave us alone, second part is indentification.